neurobooth_os/mock/mock_server_stm.py

```diff
@@ -39,18 +39,18 @@ def mock_stm_routine(host, port, conn):
     for data, connx in get_client_messages(s1, port=port, host=host):
 
         if "prepare" in data:
-            # data = "prepare:collection_id:str(tech_obs_log_dict)"
+            # data = "prepare:collection_id:str(log_task_dict)"
 
             collection_id = data.split(":")[1]
-            tech_obs_log = eval(data.replace(f"prepare:{collection_id}:", ""))
-            subject_id_date = tech_obs_log["subject_id-date"]
+            log_task = eval(data.replace(f"prepare:{collection_id}:", ""))
+            subject_id_date = log_task["subject_id-date"]
 
-            ses_folder = f"{config.paths['data_out']}{subject_id_date}"
+            ses_folder = os.path.join(config.paths['data_out'], subject_id_date)
```

Comment:
@AdoNunes always use `os.path.join` instead of joining by string formatting. It's more robust to differences between operating system